app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -95,18 +95,28 @@ <h2 class="heading-medium" id="howTo">@messages("landingPage.howTo.h2")</h2>
     <ul class="list-bullet">
         <li>@messages("landingPage.howTo.list1.bullet1")</li>
         <li>@messages("landingPage.howTo.list1.bullet2")</li>
-        <li>@messages("landingPage.howTo.list1.bullet3")</li>
+
+        <li>
+            @if(appConfig.amendReturnsEnabled){
+                messages(landingPage.amendReturn.howTo.list1.bullet3")
+            }else{
+                @messages("landingPage.howTo.list1.bullet3")
+            }
+        </li>
     </ul>
 
-    <div class="panel panel-border-wide">
-        <p>
-            @if(appConfig.furtherReturnsEnabled){
+    @if(appConfig.amendReturnsEnabled){
+    }else{
```

Comment:
you don't need an `if-else` here - couldn't you just have `if(!appConfig.amendReturnsEnabled))`?

Reply:
Are you looking for this please?
```
    @if(!appConfig.amendReturnsEnabled){
        <div class="panel panel-border-wide">
            <p>
                @if(appConfig.furtherReturnsEnabled){
                @Html(messages("landingPage.howTo.furtherReturns", appConfig.contactHmrc))
                } else{
                @Html(messages("landingPage.howTo.firstReturnsOnly", appConfig.contactHmrc))
                }
            </p>
        </div>
    }
```

Reply:
yeah, there's no point having an empty branch in an if-else statement

Reply:
Sorry Initially I did same thing and confused with your 2nd comment
`can you ever reach this statement with !appConfig.amendReturnsEnabled above?
`